neocore/Cryptography/ECCurve.py

```diff
@@ -570,11 +602,16 @@ def decompress(self, x, flag):
         return self.point(x, ysquare.sqrt(flag))
 
     def decode_from_reader(self, reader):
-
-        f = reader.ReadByte()
-
-        if f == 0:
+        """
+        Raises:
+            NotImplementedError: if the read int.from_bytes `f` == 4, 6, or 7
```

Comment:
```suggestion
            NotImplementedError: if the an unsupported point encoding is used
```

Reply:
just tweaked a little to "if an unsupported point encoding is used"